src/libstd/sys/unix/process/magenta.rs

```diff
@@ -119,21 +119,14 @@ extern {
                               avail: *mut mx_size_t) -> mx_status_t;
 }
 
-// Handle Info entries associate a type and optional
-// argument with each handle included in the process
-// arguments message.
-pub fn mx_hnd_info(hnd_type: u32, arg: u32) -> u32 {
-    (hnd_type & 0xFFFF) | ((arg & 0xFFFF) << 16)
-}
-
-extern {
-    pub fn mxio_get_startup_handle(id: u32) -> mx_handle_t;
+pub fn mx_job_default() -> mx_handle_t {
+    unsafe { return __magenta_job_default; }
```

Comment:
Nit: Just use `unsafe { __magenta_job_default }`. Idiomatic Rust only uses `return` for early returns.